clean_project/test_data/config_file.yaml

```diff
@@ -0,0 +1,2 @@
+{'files_path': ['test_data/full_file_dirty.txt']
```

Comment:
for reference: here is how a yaml file should look like: https://en.wikipedia.org/wiki/YAML

i'm not even sure how you're making it run with this format